src/templates/_macros/rating.html

```diff
@@ -1,41 +1,40 @@
 {% include "_macros/stars.html" %}
 
 {% macro rating(this, detailpage=false, current_version=None) %}
-<li data-report-uri="{{ this.report_spam }}" data-rating="{{ this.rating }}"
-    class="review{{ ' flagged' if this.is_flagged }} c"
+<li class="review {{ 'flagged' if this.is_flagged }} c"
```

Comment:
Not a fan of this because of the extra whitespace though we know the parser doesn't care.